apidef/oas/server_test.go

```diff
@@ -281,12 +279,17 @@ func TestTagsExportServer(t *testing.T) {
 				TagsDisabled: true,
 				Tags:         []string{"a", "b", "c"},
 			},
-			nil,
+			&GatewayTags{
+				Enabled: false,
+				Tags:    []string{"a", "b", "c"},
+			},
 		},
 		{
 			"empty segment tags",
 			apidef.APIDefinition{},
-			nil,
+			&GatewayTags{
+				Enabled: true,
+			},
```

Comment:
@furkansenharputlu this seems suspect - it's an invalid OAS definition, we should allocate Tags:[], but it's now still an invalid api definition with the minItems:1 validation - should we maybe add a default value?